src/app/containers/CpsAssetPageMain/index.test.jsx

```diff
@@ -11,8 +11,9 @@ import { RequestContextProvider } from '#contexts/RequestContext';
 import { ToggleContext } from '#contexts/ToggleContext';
 import CpsAssetPageMain from '.';
 import preprocessor from '#lib/utilities/preprocessor';
-import igboPageData from '#data/igbo/cpsAssets/afirika-23252735';
-import pidginPageData from '#data/pidgin/cpsAssets/23248703';
+import styIgboPageData from '#data/igbo/cpsAssets/afirika-23252735';
```

Comment:
Just a quick thought. Do we need to specify service names for the different pieces of asset page data? It feels like it might be specifically testing the CPS asset data from these services which from my understand we're testing in a general sense.

Reply:
Do you mean in the variable names? If so, I'd say not, clear them out and make it clearer :)